RELEASE/scripts/autoscend.ash

```diff
@@ -2769,6 +2769,11 @@ void auto_begin()
 	backupSetting("logPreferenceChange", "true");
 	backupSetting("logPreferenceChangeFilter", "maximizerMRUList,testudinalTeachings,auto_maximize_current");
 	backupSetting("maximizerMRUSize", 0); // shuts the maximizer spam up!
+
+	string userForbidden = get_property("forbiddenStores");
+	if (!userForbidden.contains_text("3408540")) {
+		backupSetting("forbiddenStores", userForbidden + ",3408540"); // forbid Dance Police
```

Comment:
is the leading comma going to be fine if it currently is "" or will it need to check and have a version without the comma if == "" and with comma on an else?

Reply:
the leading comma works fine with an empty string. It's what I tested with and it's what garbo uses too IIRC.

Reply:
thanks. well in that case looks good to me